### ISSUE TITLE
cannot connect to the local host jabber server 

### ISSUE THREAD

**Issue by jitu89** [processone/demo-xmpp-ios#3]
Hi,
I'm try to log in to the local server [192.168.1.32] but can't connect it.
This is the Log i am getting

2016-06-17 13:24:38:360 ChatApp[516:2a07] SEND: <?xml version='1.0'?>
2016-06-17 13:24:38:366 ChatApp[516:2a07] SEND: <stream:stream xmlns='jabber:client' xmlns:stream='http://etherx.jabber.org/streams' version='1.0' to='192.168.1.32'>
2016-06-17 13:24:38:430 ChatApp[516:3d03] RECV: <stream:stream xmlns="jabber:client" xmlns:stream="http://etherx.jabber.org/streams" id="2612510292109683183" from="192.168.1.32" version="1.0" stream1:lang="en"/>
2016-06-17 13:24:38:431 ChatApp[516:3d03] RECV: <stream:features xmlns:stream="http://etherx.jabber.org/streams"><c xmlns="http://jabber.org/protocol/caps" hash="sha-1" node="http://www.process-one.net/en/ejabberd/" ver="1h0AkH4ZFdRE+5QKmkJjNRZy5yQ="/><starttls xmlns="urn:ietf:params:xml:ns:xmpp-tls"><required/></starttls>/stream:features
2016-06-17 13:24:38:432 ChatApp[516:3d03] SEND: <starttls xmlns='urn:ietf:params:xml:ns:xmpp-tls'/>

**Comment by baoluo** [comment]
Hello @jitu89 did you find the problem ?

**Comment by jitu89** [comment]
Yes the problem was with the https protocal while we try to connect.

*Regards, *
*Jitendra Rai *
